### PR TITLE
UI - hosts step 2 first cut

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/24639

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-345-8y5kx25x.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-345-k095g05y.web.app

File: strr-host-pm-web/app/components/summary/Owners.vue [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        <!-- <UDropdown :items="[{ slot: 'remove' }]" :disabled="expand.openedRows.length" :ui="{ width }">
+          <UButton icon="i-mdi-chevron-down" variant="link" />
+          <template #item="{ item }">
+            <UButton :label="$t('word.remove')" icon="i-mdi-trashcan" @click="ownerStore.removeHostOwner(index)" />
+          </template>
+        </UDropdown> -->
+        <UPopover :popper="{ placement: 'bottom-end' }">
+          <UButton
+            icon="i-mdi-menu-down"
+            aria-label="show more options"
```
Comment (deetz99) [inline strr-host-pm-web/app/components/summary/Owners.vue:155]: needs i18n

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-345-8y5kx25x.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-345-k095g05y.web.app

File: strr-host-pm-web/nuxt.config.ts [hunk 2]
```diff
@@ -39,8 +39,8 @@ export default defineNuxtConfig({
   },
 
   extends: [
-    // '../strr-base-web' // dev only
-    ['github:bcgov/STRR/strr-base-web', { install: true }]
+    '../strr-base-web' // dev only
+    // ['github:bcgov/STRR/strr-base-web', { install: true }]
```
Comment (deetz99) [inline strr-host-pm-web/nuxt.config.ts]: does this need to be switched?

**Review by deetz99 — APPROVED** [review]
Looks good to me 👍 nice work!

File: strr-host-pm-web/nuxt.config.ts (hunk 2 above)
Comment (deetz99) [reply strr-host-pm-web/nuxt.config.ts]: I see you've already fixed this so nvm

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-345-k095g05y.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-345-8y5kx25x.web.app

**Pull request merged**